config/locales/en.yml

```diff
@@ -1276,9 +1276,9 @@ en:
     open_in_web: Open in web
     over_character_limit: character limit of %{max} exceeded
     pin_errors:
+      direct: Direct posts cannot be pinned
```

Comment:
Direct posts is not a term we use as such anywhere else. Direct-visibility posts might be more clear. Or mention-only posts… to be honest maybe you can keep “non-public”, it still works and sounds better than direct posts.